assets/compat/btc-rpc-proxy.rules.yaml

```diff
@@ -0,0 +1,67 @@
+          name: c-lightning
+          allowed-calls: []
+    - SET:
+        var: 'users.[first(item => ''item.name = "c-lightning")].password'
+        to-entropy:
+          charset: "a-z,A-Z,0-9"
+          len: 22
+- rule: '''users.[first(item => ''item.name = "c-lightning")].allowed-calls.* = "echo"'
+  description: 'RPC user "c-lightning" must have "echo" enabled'
+  suggestions:
+    - PUSH:
+        to: 'users.[first(item => ''item.name = "c-lightning")].allowed-calls'
+        value: "echo"
+- rule: '''users.[first(item => ''item.name = "c-lightning")].allowed-calls.* = "gettxout"'
+  description: 'RPC user "c-lightning" must have "gettxout" enabled'
+  suggestions:
+    - PUSH:
+        to: 'users.[first(item => ''item.name = "c-lightning")].allowed-calls'
+        value: "gettxout"
+- rule: '''users.[first(item => ''item.name = "c-lightning")].allowed-calls.* = "getblockchaininfo"'
+  description: 'RPC user "c-lightning" must have "getblockchaininfo" enabled'
+  suggestions:
+    - PUSH:
+        to: 'users.[first(item => ''item.name = "c-lightning")].allowed-calls'
+        value: "getblockchaininfo"
+- rule: '''users.[first(item => ''item.name = "c-lightning")].allowed-calls.* = "sendrawtransaction"'
+  description: 'RPC user "c-lightning" must have "sendrawtransaction" enabled'
+  suggestions:
+    - PUSH:
+        to: 'users.[first(item => ''item.name = "c-lightning")].allowed-calls'
+        value: "sendrawtransaction"
+- rule: '''users.[first(item => ''item.name = "c-lightning")].allowed-calls.* = "getblockhash"'
+  description: 'RPC user "c-lightning" must have "getblockhash" enabled'
+  suggestions:
+    - PUSH:
+        to: 'users.[first(item => ''item.name = "c-lightning")].allowed-calls'
+        value: "getblockhash"
+- rule: '''users.[first(item => ''item.name = "c-lightning")].allowed-calls.* = "getblock"'
+  description: 'RPC user "c-lightning" must have "getblock" enabled'
+  suggestions:
+    - PUSH:
+        to: 'users.[first(item => ''item.name = "c-lightning")].allowed-calls'
+        value: "getblock"
+- rule: '''users.[first(item => ''item.name = "c-lightning")].allowed-calls.* = "estimatesmartfee"'
+  description: 'RPC user "c-lightning" must have "estimatesmartfee" enabled'
+  suggestions:
+    - PUSH:
+        to: 'users.[first(item => ''item.name = "c-lightning")].allowed-calls'
+        value: "estimatesmartfee"
+- rule: '''users.[first(item => ''item.name = "c-lightning")].allowed-calls.* = "getnetworkinfo"'
+  description: 'RPC user "c-lightning" must have "getnetworkinfo" enabled'
+  suggestions:
+    - PUSH:
+        to: 'users.[first(item => ''item.name = "c-lightning")].allowed-calls'
+        value: "getnetworkinfo"
+- rule: 'users.[first(item => ''item.name = "c-lightning")].fetch-blocks?'
+  description: 'RPC user "c-lightning" must have "Fetch Blocks" enabled'
+  suggestions:
+    - SET:
+        var: 'users.[first(item => ''item.name = "c-lightning")].fetch-blocks'
+        to-value: true
```

Comment:
has it been verified these attributes are actually set in proxy?

Reply:
If you mean rpc commands in general, yes they are working as expected. If you mean "fetch-blocks", no, I haven't tested that one specifically.